react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -145,6 +145,18 @@ type Props = {
     transport: Array<Object>
 };
 
+/**
+ * Click handler.
+ *
+ * @param {SyntheticEvent} event - The click event.
+ * @returns {void}
+ */
+function onClick(event) {
+    // If the event is propagated to the thumbnail container the participant will be pinned. That's why the propagation
```

Comment:
Can't this be avoided by setting `pointer-events` to `none` ?

Reply:
From the description here (https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events#values) I don't think  we can. But actually even if it was possible I kind of prefer it this way. It is easier for me to miss a css prop :)

We can kind of optimise this at some point and maybe we won't need this. The problem is that the ConnectionStatsTable is part of the thumbnails DOM and ReactDom... Maybe at some point we can move it to the `<body>`. Technically it doesn't need to be part of the thumbnail at all.

Reply:
ack